internal/cmd/ssh.go

```diff
@@ -263,7 +263,6 @@ func provisionSSHKey(ctx context.Context, opts provisionSSHKeyOptions) (string,
 		}
 
 		if userPublicKeyContains(opts.user.PublicKeys, existing.PublicKeyID) {
-			// TODO: check expiration when expiry is added
```

Comment:
This was no longer necessary, because the API simply doesn't return expired keys. That way we don't have to handle them in every client.